plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -52,47 +52,27 @@ protected QueryRunner createQueryRunner()
         return createAccumuloQueryRunner(ImmutableMap.of());
     }
 
-    @SuppressWarnings("DuplicateBranchesInSwitch")
     @Override
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
-        switch (connectorBehavior) {
-            case SUPPORTS_DELETE:
-            case SUPPORTS_UPDATE:
-            case SUPPORTS_MERGE:
-                return false;
-
-            case SUPPORTS_TOPN_PUSHDOWN:
-                return false;
-
-            case SUPPORTS_RENAME_SCHEMA:
-            case SUPPORTS_DROP_SCHEMA_CASCADE:
-                return false;
-
-            case SUPPORTS_CREATE_TABLE_WITH_TABLE_COMMENT:
-            case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
-                return false;
-
-            case SUPPORTS_ADD_COLUMN:
-            case SUPPORTS_SET_COLUMN_TYPE:
-                return false;
-
-            case SUPPORTS_COMMENT_ON_TABLE:
-            case SUPPORTS_COMMENT_ON_COLUMN:
-                return false;
-
-            case SUPPORTS_CREATE_MATERIALIZED_VIEW:
-                return false;
-
-            case SUPPORTS_NOT_NULL_CONSTRAINT:
-                return false;
-
-            case SUPPORTS_ROW_TYPE:
-                return false;
```

Comment:
These cases are separated intentionally.

Reply:
What's the criteria for such separation? Looking at each of those tests classes, the separation doesn't seem to provide any signal compared to the alternative. Also, the way they are structured currently it forces adding that suppression at the top, which is a code smell.

Reply:
I agree.  As a new reader of this code, the old organization didn't mean anything to me, and I find the new one more clear.  There is clearly a block supported and unsupported things.

Reply:
Please remove or update javadoc of `hasBehavior` method in `BaseConnectorTest` and `BaseConnectorSmokeTest`:
```
    /**
     * Make sure to group related behaviours together in the order and grouping they are declared in {@link TestingConnectorBehavior}.
     * If required, annotate the method with {@code @SuppressWarnings("DuplicateBranchesInSwitch")}.
     */
```